XenModel/Alerts/PolicyAlert.cs

```diff
@@ -177,15 +177,10 @@ public string ShortFormatBody
                             else
                             {
                                 return string.Format(Messages.VM_SNAPSHOT_SCHEDULE_FAILED, Message.FriendlyName(XenAPI.Message.MessageType.VMSS_SNAPSHOT_FAILED.ToString()), numberOfVMsFailed);
-                            }
-                                
+                            }      

```

Comment:
Is the PolicyType field still needed? The relevant MessageTypes are still exposed via the API, but if they won't occur anymore, do we need to construct a different PolicyAlert?